backend/src/api/VoteController.ts

```diff
@@ -76,6 +72,19 @@ export default class VoteController {
             const max = type === 'user' ? 2 : 1;
             const rangedVote = Math.max(Math.min(vote, max), -max);
 
+            const targetUserId = await this.voteManager.getUserIdByVote(id, type);
+            if (targetUserId && type !== 'user') {
+                const voteKey = `${userId}:${targetUserId}`;
+                try {
+                    await this.voteDailyRateLimiter.consume(voteKey);
+                    if (rangedVote < 0) {
```

Comment:
But why only for minuses? mass-plusing will have same effect on regulation

Reply:
> mass-plusing will have same effect on regulation

You mean zero effect?

Reply:
Yup - so why treat them differently

Reply:
Solely because people are using them differently and complaining only about minuses.